src/main/kotlin/study/racingcar/io/OutputView.kt

```diff
@@ -0,0 +1,39 @@
+package study.racingcar.io
+
+import study.racingcar.racing.RacingResult
+import study.racingcar.racing.Record
+
+object OutputView {
+    fun printResult(result: RacingResult) {
+        result.rounds
+            .forEach { round: List<Record> ->
+                printByRound(round)
+                printEndOfTurn()
+            }
+    }
+
+    fun printWinners(result: RacingResult) {
+        val winnersName = result.winners()
+            .joinToString(",") { winner ->
+                winner.name
+            }
```

Comment:
```suggestion
            joinToString(",") { it.name }
```
어느 정도 취향 차가 있을 수 있지만 크게 헷갈리지 않는다면 `it` 키워드를 많이 사용하면 좀 더 간단하게 코드를 작성할 수 있습니다.